### ISSUE TITLE
Compiled on openSUSE but it wont launch with GTK 3 error.

### ISSUE THREAD

**Issue by Skrappjaw** [jack-mixer/jack_mixer#165]
Just moved to openSUSE and the only piece of software I cant get through the repo is jack-mixer.  I compiled everything as instructed and installed successfully.  I checked yast to see if gtk 3 was installed and it is.
```
jack_mixer
Traceback (most recent call last):
  File "/usr/bin/jack_mixer", line 5, in <module>
    from jack_mixer.app import main
  File "/usr/lib/python3.10/site-packages/jack_mixer/app.py", line 23, in <module>
    gi.require_version("Gtk", "3.0")
  File "/usr/lib64/python3.10/site-packages/gi/__init__.py", line 126, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Gtk not available

```

**Comment by SpotlightKid** [comment]
Are you sure you've got `pygobject` and `pycairo` installed?

Can you please post the output of `meson builddir --prefix=/usr --buildtype=release` here? But make sure to remove `builddir` first.

**Comment by Skrappjaw** [comment]
Sorry i didnt get back to you earlier. I re ran the compiler and it worked. Now I'm trying to build this again on a new system and have no idea why it resolved or why im getting the same error again. 

```
jack_mixer
Traceback (most recent call last):
  File "/usr/bin/jack_mixer", line 5, in <module>
    from jack_mixer.app import main
  File "/usr/lib64/python3.11/site-packages/jack_mixer/app.py", line 23, in <module>
    gi.require_version("Gtk", "3.0")
  File "/usr/lib64/python3.11/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace Gtk not available for version 3.0

```

here is the output you asked for.
```
meson setup --reconfigure builddir
The Meson build system
Version: 1.2.3
Source dir: /home/skrapp/jack_mixer
Build dir: /home/skrapp/jack_mixer/builddir
Build type: native build
Project name: jack_mixer
Project version: 18
C compiler for the host machine: cc (gcc 13.2.1 "cc (SUSE Linux) 13.2.1 20230912 [revision b96e66fd4ef3e36983969fb8cdd1956f551a074b]")
C linker for the host machine: cc ld.bfd 2.41.0.20230908-1
Host machine cpu family: x86_64
Host machine cpu: x86_64
Dependency glib-2.0 found: YES 2.78.1 (cached)
Library m found: YES
Dependency jack found: YES 1.9.22 (cached)
Dependency jack found: YES 1.9.22 (cached)
Program python3 (gi, cairo, appdirs) found: YES (/usr/bin/python3) modules: gi, cairo, appdirs
Program cython found: YES (/home/skrapp/.local/bin/cython)
Configuring version.py using configuration
Program msgfmt found: YES (/usr/bin/msgfmt)
Configuring jack_mixer-de.mo with command
Configuring jack_mixer-es.mo with command
Configuring jack_mixer-fr.mo with command
Program rst2man.py found: YES (/home/skrapp/.local/bin/rst2man.py)
Configuring jack_mix_box.1.rst using configuration
Configuring jack_mixer.1.rst using configuration
Build targets in project: 5

jack_mixer 18

  Configuration
    Build jack_mixer GUI    : True
    JACK MIDI support       : True
    Debug messages (verbose): False
    Build for wheel         : False

  User defined options
    buildtype               : release
    prefix                  : /usr

Found ninja-1.11.1 at /usr/bin/ninja
Cleaning... 0 files.
```

**Comment by Skrappjaw** [comment]
I figured it out. Must have gtk3-devel package installed on opensuse
